packages/cli/src/bin/ceramic.ts

```diff
@@ -9,7 +9,7 @@ program
     .option('--anchor-service-api <url>', 'The anchor service URL to use')
     .option('--validate-docs', 'Validate documents according to their schemas. It is enabled by default')
     .option('--pinning <url...>', 'Pinning endpoints')
-    .option('--pinning-store-path <url>', `The directory path used for pinning service. Defaults to WORKING_DIR/${DEFAULT_PINNING_STORE_PATH}`)
+    .option('--pinning-store-path <url>', `The directory path used for pinning service. Defaults to WORKING_DIR`)
```

Comment:
Don't think this should say `<url>`. A path is not a uri afaik. Would also prefer to rename this to something like `pinset-directory`.